lib/bootloader_setup.pm

```diff
@@ -612,7 +612,7 @@ sub bootmenu_default_params {
         }
         push @params, "Y2DEBUG=1";
     }
-    elsif (get_var('AGAMA')) {
+    elsif (get_var('AGAMA') && !is_ppc64le()) {
```

Comment:
I don't think we need to enter this function in agama `bootmenu_default_params`, we could skip it from bootloader_pvm.pm with `unless ...` seems that it is only though for qemu and it is not our implementation.

Reply:
That is what I mentioned yesterday, we will need to tricky some calls and steps to make Agama boot in a clean way. Then, it's better to do another approach. In any case, on my way to this skip.

Reply:
If I add unless in the `bootloader_pvm.pm` file we will skip this call for the current AGAMA call I guess from another team, is that ok?

Reply:
Take a look at 3d304e9d6

Reply:
well, put a diplomatic message like this one on top of the skip:
`skipping this setup as trigger general code for openSUSE that breaks powerVM scenario`

Reply:
090ec21b8